tests/foreman/ui_airgun/test_hardwaremodel.py

```diff
@@ -0,0 +1,78 @@
+"""Test class for Hardware Model UI
+
+:Requirement: HardwareModel
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: Low
+
+:Upstream: No
+"""
+from fauxfactory import gen_string
+from nailgun import entities
+from pytest import raises
+
+from robottelo.decorators import tier2, upgrade
+from tests.foreman.ui_airgun.test_host import create_fake_host
```

Comment:
Instead of using create_fake_host from another test, why don't we keep it in the commonplace and use it from there?  What do you think?

Reply:
Yeah, it can be better place for that